lib/apiservers/engine/backends/container.go

```diff
@@ -813,13 +813,13 @@ func (c *Container) ContainerWait(name string, timeout time.Duration) (int, erro
 	}
 
 	containerInfo := results.Payload
-
+	started := time.Unix(*containerInfo.ProcessConfig.StartTime, 0)
```

Comment:
do we need to check containerInfo != nil && containerInfo.ProcessConfig != nil here?